src/server/mod.rs

```diff
@@ -416,7 +413,7 @@ impl LanguageServer for LspServer {
                     true,
                 )),
                 execute_command_provider: Some(lsp::ExecuteCommandOptions {
-                    commands: vec![LspServerCommand::InjectTagFilter.into(), LspServerCommand::InjectTagValueFilter.into(), LspServerCommand::InjectFieldFilter.into(), LspServerCommand::InjectMeasurementFilter.into()],
+                    commands: vec![],
```

Comment:
Hm, this would be a bug: we need to have all the accepted commands in this list. That the server even responds to the commands that aren't included here is likely a bug in `lspower`. I don't think it need to be done in this patch, since it's not breaking anything, but can you file an issue about this?

Reply:
Sure! Here is the issue created for this bug #587